### DISCUSSION TITLE
無人機救援、無人船清垃圾 高雄港、安平亞果碼頭明年初試驗

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1803] [General]
自主移動機器人雖然已比較常見，但要能在碼頭進行運輸服務，除了必須具備自動偵測及避障功能，也要克服海港戶外環境；虛擬體驗則是利用VR體驗海港智慧巡檢任務及豪華遊艇導覽。

文章中提出對於碼頭運輸、油污監測、海面清潔及水深量測、落水救援等層面，若單靠人工巡查，耗時且危險性高。

運用無人機，如果接獲人員溺水通報，無人機會吊掛救生圈，結合定位技術及視距外遠端操控，確認案發位置後，投放救生圈給溺水者。
運用無人船清除垃圾模式，具備避障機制，可清理垃圾、油污，以及量測水深。

資料來源：https://news.ltn.com.tw/news/life/breakingnews/4504347

應用這些高科技機器來代替人員的危險性工作，不但可以提高安全性還能更有效率的完成，因住在港口附近，很常看到海上有許多垃圾以及油污，如果可以夠透過無人船來清理，可以使海洋環境更加乾淨！

**Comment by yuchien-lin** [comment]
我覺得這種高科技下的產物，讓我感到時代的差別，在以前的時代根本不可能有這種這麼便利的科技，然而這個高科技的時代下讓我感到了沒有什麼是不可能，如同chatgpt的橫空出世，這類的新科技未來只會越來越多。

**Comment by B1nn9** [comment]
希望除了港口附近外，未來無人船清運垃圾的範圍能擴展到汪洋中的海漂垃圾，根據統計，在汪洋中每五分鐘就出現大約20件人為廢棄物，所以海岸邊的垃圾相較之下也只是冰山一角，希望有朝一日高科技的發展能順利解決海洋垃圾問題。

**Comment by eric202501** [comment]
這確實是一個具潛力的應用領域，利用自主移動機器人在碼頭和海港碼頭和海港進行各種任務。這種科技的應用不僅提高了工作效率，還能夠降低工作人員面臨的風險。特別是在海港環境中，自主移動機器人的自動偵測和避障功能，以及無人機的應用，能夠在短時間內完成各種任務。這對於保護海洋環境、提升碼頭運輸效率以及增加安全性都是十分有效的。

**Comment by Ryanlolry** [comment]
這些自主移動機器人和無人機的應用似乎在解決一些需要長時間、高風險、甚至危險的工作任務上具有潛在的效益。例如，無人機可以接收溺水通報，配備救生圈並精確投放至溺水者處。無人船也能清除垃圾和油污，同時進行水深量測。這些技術和應用可能有助於提高效率、降低風險，同時改善碼頭和海港環境的清潔度和安全性。

**Comment by chuanmin0317** [comment]
這些自主移動機器人或是AI的運用可以幫助我們解決較為困難、重複性或是危險的問題，讓我們可以將自己的時間放在具有創造力也更喜歡的事情上

**Comment by FakeWood** [comment]
用這些高科技機器來做比較危險的工作真是太棒了，這樣就不會有工作人員受傷的問題，還可以更有效率，希望其他比較危險的工作也可以靠無人機來達成。
